phpstan-baseline.neon

```diff
@@ -5178,6 +5178,11 @@ parameters:
 			count: 1
 			path: app/Utils/TestCreator.php
 
+		-
+			message: "#^Parameter \\#1 \\$data of function imagecreatefromstring expects string, string\\|false given\\.$#"
+			count: 1
+			path: app/Utils/TestCreator.php
```

Comment:
This can easily be fixed by handling the `false` return value from `imagecreatefromstring`, right?